.pre-commit-config.yaml

```diff
@@ -1,10 +1,4 @@
 repos:
-- repo: https://github.com/asottile/add-trailing-comma
-  rev: v0.7.0
-  hooks:
-  - id: add-trailing-comma
-    # TODO: Remove this line. For now, we only format the api/ directory
-    files: ^api/
 - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
@Johnetordoff - if i'm reading this right, this PR isn't doing what you're saying it's doing.   Pre-commit-hooks are already in place.  It looks to me like you're just reducing the trailing comma checks in the API directory, you haven't expanded coverage anywhere.